src/definitions/elements/label.less

```diff
@@ -90,21 +90,37 @@ a.ui.label {
   margin: 0 @detailIconDistance 0 0;
 }
 
-
-/* Removable label */
+/* Removable and Hoverable label */
 .ui.label > .close.icon,
-.ui.label > .delete.icon {
+.ui.label > .delete.icon,
+.ui.label > .hoverable.icon {
   cursor: pointer;
-  margin-right: 0;
-  margin-left: @deleteMargin;
   font-size: @deleteSize;
   opacity: @deleteOpacity;
   transition: @deleteTransition;
 }
-.ui.label > .delete.icon:hover {
+
+/* Backward compatible positioning */
+.ui.label:not(.icon) > .delete.icon {
```

Comment:
`.ui.label:not(.icon) > .close.icon,` support missing

Reply:
Added.